be/src/exec/pipeline/scan/olap_chunk_source.cpp

```diff
@@ -258,6 +259,22 @@ Status OlapChunkSource::_init_unused_output_columns(const std::vector<std::strin
     return Status::OK();
 }
 
+Status OlapChunkSource::_init_column_access_paths(Schema* schema) {
+    // column access paths
+    auto paths = _scan_ctx->column_access_paths();
```

Comment:
would it be nullptr?

Reply:
may be

Reply:
maybe auto*

Reply:
c++ can derive auto is pointer, but better to let it explicitly, I will modify